tests/Objects/RequiredPropertyTest.php

```diff
@@ -162,4 +162,29 @@ public function requiredStringPropertyDataProvider(): array
             ],
         );
     }
+
+    /**
+     * @dataProvider requiredAndOptionalPropertiesDataProvider
+     *
+     * @throws FileSystemException
+     * @throws RenderException
+     * @throws SchemaException
+     */
+    public function testProvidedConstPropertiesIsValid(string $reqPropertyValue, ?string $optPropertyValue): void
```

Comment:
I'd move this test also to the ConstPropertyTest. Additionally, there should be a test present, which only provides the `requiredProperty`.

Reply:
I agree with you, I moved and improved the test